templates/UmbracoPackage/.template.config/template.json

```diff
@@ -41,9 +41,16 @@
       "description": "The version of Umbraco.Cms to add as PackageReference.",
       "type": "parameter",
       "datatype": "string",
-      "defaultValue": "10.0.0-rc1",
+      "defaultValue": "11.0.0",
```

Comment:
Once PR https://github.com/umbraco/Umbraco-CMS/pull/13481 is merged, we won't need to change this value manually anymore 😄